clients/spas/Dockerfile

```
@@ -23,7 +23,7 @@ RUN npm install @rollup/rollup-linux-x64-gnu || true
 
 # Copy source files
 COPY ./specs/apis/ ./specs/apis/
-COPY --exclude=**/*.test.ts --exclude=vitest.config.ts ./clients/ ./clients/
+COPY --exclude=**/*.test.ts --exclude=vitest.config.ts ./clients/spas/ ./clients/spas/
```

Hadolint (2.12.0)

[error] invalid flag: --exclude

(DL1000)